EvaluableExpression.go

```diff
@@ -746,144 +753,3 @@ func (this EvaluableExpression) String() string {
 
 	return this.inputExpression
 }
-
-func sanitizeParamters(parameters map[string]interface{}) (map[string]interface{}, error) {
-
-	var ret map[string]interface{}
-	var needsSanitization bool
-	var err error
-
-	if parameters == nil {
-		return DUMMY_PARAMETERS, nil
-	}
-
-	// we don't copy anything unless there is something that needs to be sanitized.
-	needsSanitization = false
-
-	for key, value := range parameters {
-
-		// make sure that the parameter is a valid type.
-		err = checkValidType(key, value)
-		if err != nil {
-			return nil, err
-		}
-
-		// should be converted to fixed point?
-		if isFixedPoint(value) {
-
-			// sanitize.
-			// if we haven't yet made a new map, do so and copy all keys.
-			if !needsSanitization {
-
-				ret = make(map[string]interface{}, len(parameters))
-
-				for innerKey, innerValue := range parameters {
-					ret[innerKey] = innerValue
-				}
-
-				needsSanitization = true
-			}
-
-			ret[key] = castFixedPoint(value)
-		}
-	}
-
-	if needsSanitization {
-		return ret, nil
-	}
-	return parameters, nil
-}
-
-func checkValidType(key string, value interface{}) error {
```

Comment:
This and the below all moved into the `parameters.go` file.  The logic didn't change (except for getting rid of the sanitizeParameters method).